DynamoDB/CreateTable.js

```diff
@@ -0,0 +1,68 @@
+var AWS = require("aws-sdk");
+var keys = require("../keys.json");
+
+
+AWS.config.update({
+  region: keys.region,
+  endpoint: keys.endpoint
+});
+
+var dynamodb = new AWS.DynamoDB();
+
+var params = {
+    TableName : "V-Transfer",
+    KeySchema: [       
+        { AttributeName: "PK", KeyType: "HASH"},  //Partition key
```

Comment:
I think we should be using the names of our partition keys as defined in the data model.